README.md

```diff
@@ -5,7 +5,7 @@
 ### Summary
 PyScript is a Pythonic alternative to Scratch, JSFiddle or other "easy to use" programming frameworks, making the web a friendly, hackable, place where anyone can author interesting and interactive applications.
 
-To get started see [GETTING-STARTED](GETTING-STARTED.md).
+To get started see the [getting started tutorial](docs/tutorials/getting-started.md).
```

Comment:
These introductory phrases: "To get started" and "For examples" need commas after them.